src/react_native/share.cljs

```diff
@@ -3,8 +3,10 @@
     ["react-native-share" :default react-native-share]))
 
 (defn open
-  [options on-success on-error]
-  (-> ^js react-native-share
-      (.open (clj->js options))
-      (.then on-success)
-      (.catch on-error)))
+  ([options]
+   (open options #() #()))
```

Comment:
Minor, maybe passing `nil` instead of empty functions? IDK if that would work, but imo is cleaner if works

Reply:
if we try to call `nil` as function we get an exception, so I think empty funcitons are better.